projects/js-packages/components/components/status/style.module.scss

```diff
@@ -0,0 +1,63 @@
+.status {
+    --font-size: var( --font-body-extra-small ); // 12px
+    --indicator-size: calc( var( --font-size ) * 0.666 ); // 8px
```

Comment:
It is good to explain where this number came from. Looks like a magic number.

Reply:
Updated to use the `Text` component and `em`s to determine the indicator size, so this is resolved by way of being removed 😁

https://github.com/Automattic/jetpack/pull/30209/commits/0b7c845d0def850fe5099d654fe2371ad25a17b2